### PR TITLE
Adds examples to readme

### PR DESCRIPTION
**Pull request by Ross-Hunter**
I had to do additional searching to find a blog post to explain how to use this gem, so I put some quick examples in the readme to make it easier for people to pick up how to use sms-spec.

### CODE REVIEW

File: README.markdown [hunk 1]
```diff
@@ -30,7 +30,26 @@ Spec::Runner.configure do |config|
   config.include(SmsSpec::Matchers)
 end
 
-SmsSpec.driver = :twilio-ruby #this can be any available sms-spec driver
+SmsSpec.driver = :'twilio-ruby' #this can be any available sms-spec driver
```
Comment (manlycode) [inline README.markdown]: Prefer double-quotes.

File: README.markdown [hunk 2]
```diff
@@ -30,7 +30,26 @@ Spec::Runner.configure do |config|
...
+```
+
+### Usage
+
+```ruby
+describe 'alarm notifications' do
```
Comment (manlycode) [inline README.markdown]: Use double-quotes here.

File: README.markdown [hunk 3]
```diff
@@ -30,7 +30,26 @@ Spec::Runner.configure do |config|
...
+  subject(:job) { AlarmNotificationJob.new }
+
+  context 'via sms' do
```
Comment (manlycode) [inline README.markdown]: Double quotes.

**Comment by Ross-Hunter** [comment]
💥

**Pull request merged**